playground/CustomResources/CustomResources.AppHost/TestResource.cs

```diff
@@ -34,34 +36,47 @@ public Task BeforeStartAsync(DistributedApplicationModel appModel, CancellationT
     {
         foreach (var item in appModel.Resources.OfType<TestResource>())
         {
-            if (item.TryGetLastAnnotation<CustomResourceAnnotation>(out var annotation))
+            if (item.TryGetLastAnnotation<CustomResourceAnnotation>(out var customResourceAnnotation))
             {
+                item.TryGetLastAnnotation<CustomResourceLoggerAnnotation>(out var loggerAnnotation);
+
+                if (loggerAnnotation is null)
+                {
+                    continue;
+                }
```

Comment:
Why not put both TryGetLastAnnotation in the line 39 if test?

Reply:
Can do.